specs/Deployment/DeploymentAPI.md

```diff
@@ -179,12 +186,43 @@ they shut down, to refer to the updated framework package.
     }
 ```
 
+## Repair
+
+This API performs a status check, and if the main and singleton packages are already installed at
+the required version, it will attempt to repair those packages. All information about the version,
+channel, and architecture needed are derived from the current WinAppSDK framework package.
+
+Since both the main _and_ the singleton packages may need to be repaired, it is possible that one
+package is repaired successfully but not the other. If so, the returned WindowsAppRuntimeStatus will
+contain the error, if one occurs during package install.
```

Comment:
This is unclear. There are 2 things to potentially repair (Main+Singleton). If we do Main and fail what happens? Stop and return error? Also do Singleton but then return the details of the 1st error (Main)? Or...

IOW is this a fail-on-first-error or best-effort-report-first-error?

Reply:
I copied the language from Initialize. That statement in the spec here seems to indicate best-effort-report-first error. But Initialize code behaves as fail-on-first error. The spec or the code itself needs correction to keep them both in sync. I propose to rectify the spec.

As for Repair, I think either both Initialize and Repair follow same method of fail-on-first error or best-effort-report-first-error. Hence, since Initialize follows the fail-on-first errormethod, I propose Repair should also follow the same fail-on-first error method.

Reply:
Updated the language now to reflect fail-on-first error approach.